docs/iris/src/whatsnew/1.8.rst

```diff
@@ -134,6 +134,9 @@ Deprecations
 ============
 * The original GRIB loader has been deprecated and replaced with a new
   template-based GRIB loader.
+* Deprecated the NetCDF behaviour of assigning the outermost dimension to be unlimited.
+  Switch to the new behaviour with no auto assignment by setting
+  :data:`iris.FUTURE.netcdf_no_unlimited` to True.
```

Comment:
It doesn't mention that it's for *saving* netCDF files. (And it doesn't scan particularly well either.)